app/database/models.py

```diff
@@ -32,6 +32,7 @@ class User(Base):
     telegram_id = Column(String, unique=True)
     is_active = Column(Boolean, default=False)
     language_id = Column(Integer, ForeignKey("languages.id"))
+    target_weight = Column(Float, default=None)
```

Comment:
`target_weight = Column(Float, nullable=True)`
should be instead.

Reply:
Done